ggml-impl.h

```diff
@@ -5,6 +5,7 @@
 // GGML internal header
 
 #include <assert.h>
+#include <stdlib.h>
```

Comment:
Since this file doesn't directly use stdlib.h, could we add a comment explaining why this is needed so it doesn't accidentally get removed in the future?